README.md

```diff
@@ -1 +1,11 @@
-``` shell
g++ -static lib_sample_use.cpp -std=c++11 -lfastFM -lglog -pthread -I. -L.
```
\ No newline at end of file
+Build:
+
+    > $GCC_MUSL_ROOT/x86_64-linux-musl-g++ -static -std=c++11 lib_sample_use.cpp libfastFM.a libglog.a -o foo
```

Comment:
> Be aware that, "out of the box", the wrapper **only supports C applications, not C++**. This is because the C++ libraries and headers are missing from the musl include/library path. The existing libstdc++ is actually compatible with musl in most cases and could be used by copying it into the musl library path, but the C++ header files are usually not compatible. One option may be rebuilding just libstdc++ against musl; however, **if C++ support is needed, it's recommended just to build a native toolchain targetting musl.**

Doesn't look good. I thinks it's time for me to rethink the requirements if this path it that difficult. Please wait for further updates if you don't have a better suggesting on how to proceed.